components/header.js

```diff
@@ -58,7 +58,7 @@ export default function Header() {
           <Link href="/">
             <a tabIndex="0">
               <span className="sr-only">ATP</span>
-              <img className="h-10 w-auto" src="/logo.jpg" alt="" />
+              <img className="h-10 w-auto" src="/logo.jpg" alt="" width="288" height="96" />
```

Comment:
setting image sizes is not something I have encountered before when doing web a11y work. this seems not-idiomatic, should be done with CSS if needed, and could mess with layout if we update the image files.

maybe i'm wrong! can you describe your motivation here?

Reply:
> setting image sizes is not something I have encountered before when doing web a11y work. this seems not-idiomatic, should be done with CSS if needed, and could mess with layout if we update the image files.

Yes, you are right, it isn't really an a11y thing. And about img changes, you're right about it. When the image is changed, the dimensions attribute needs to be updated.

> can you describe your motivation here?

It can greatly assist the web magnifiers and prevent layout changes on slow connections.

If this change is not required, please let me know so that I can revert this commit.

Thank you!

Reply:
Yeah, please pull the image size changes out (revert the commit or rebase and force-push the branch), and this PR will be easier to merge.

Thanks!

Reply:
> Yeah, please pull the image size changes out (revert the commit or rebase and force-push the branch), and this PR will be easier to merge.

Sure, it's done.

Thanks for the review!